common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -857,6 +857,7 @@ protected TokenResult performSilentTokenRequest(
             ((MicrosoftTokenRequest) refreshTokenRequest).setClaims(parameters.getClaimsRequestJson());
             ((MicrosoftTokenRequest) refreshTokenRequest).setClientAppName(parameters.getApplicationName());
             ((MicrosoftTokenRequest) refreshTokenRequest).setClientAppVersion(parameters.getApplicationVersion());
+            ((MicrosoftTokenRequest) refreshTokenRequest).setClientAppVersion(parameters.getCorrelationId());
```

Comment:
Why are we setting correlation id as client app version?

Reply:
That's a mistake, let me update this as Saurabh is OOO.

Reply:
Updated. @shahzaibj please re-review. Specifically: 
- if the correlation ID in the parameter object isn't set, this will explicitly set the correlation ID in the request to null. Is that a problem? Is there another place in the code that checks whether the correlation ID is null and if so sets it to something (e.g. the API layer)?
- are there other unwanted side-effects of this change?

Reply:
@SaurabhMSFT did you test this? In my tests the request is not an instance of `MicrosoftTokenRequest`, so this code is not executed.